Yubico.YubiKey/docs/users-manual/application-piv/commands.md

```diff
@@ -291,11 +291,11 @@ value and `GetData` will throw an exception.
 ___
 ## Biometric verification
 
-With biometric verification, the users can authenticate the PIV session with a successful match of a fingerprint. To be able to execute the biometric verification, the YubiKey has to have biometrics configured and must not be blocked for using biometrics. Clients can verify the conditions by reading the properties of biometric metadata (see [Get Bio metadata](#get-bio-metadata)).
+With biometric verification, users can authenticate the PIV session with a successful match of a fingerprint. To execute biometric verification, the YubiKey must have biometrics configured and enabled. Clients can verify these conditions by reading the properties of biometric metadata (see [Get Bio metadata](#get-bio-metadata)).
 
-The YubiKey keeps track of failed biometric matches and will block the biometric authentication if there are more than three such failures. In that case the client should use the PIN verification as soon as possible. The number of remaining biometric matches is returned in the command response's `AttemptsRemaining` property. The value is present only after a failed match.
+The YubiKey keeps track of failed biometric matches and will block biometric authentication if there are more than three such failures. In that case, the client should use the [PIV PIN verification](#verify) as soon as possible. The number of remaining biometric verification attempts is returned in the command response's `AttemptsRemaining` property. The value is present only after a failed match.
```

Comment:
Assuming this was referring to the PIV PIN, but please correct if wrong.

Reply:
That is correct 👍🏻

Reply:
> as soon as possible

is there a time aspect to this?

If not, perhaps it should read:

> In that case, the client must use the [PIV PIN verification](#verify) in order to unblock the Yubikey and proceed operations.